Makefile

```diff
@@ -1,3 +1,8 @@
+COMMON_GOFLAGS := -mod=vendor
```

Comment:
could you also add some documentation in the DEV readme?

Reply:
I think it's better to add the documentation once the release pipeline is in place.